docs/src/analysis/sensitivity.md

```diff
@@ -63,7 +63,10 @@ is used, i.e. going back to the AD mechanism.
 
 !!! note
 
-    ForwardDiff.jl only does forward differentiation pass through.
+    The behavior of ForwardDiff.jl is different from the other automatic differentiation libraries mentioned above.
+    The `sensealg` keyword is ignored. Instead, the differential equations are solved using `Dual` numbers for `u0` and `p`.
+    If only `p` is perturbed in the sensitivity analysis, but not `u0`, the state is still implemented as a `Dual` number.
+    ForwardDiff.jl is thus not compatible with a reverse pass through, nor with continuous forward sensitivity analysis.
```

Comment:
What do you mean by this line?

Reply:
Which line do you mean? You quoted all of them.

I do not find it obvious from the docs that ForwardDiff does not work through sensealg like the other AD libraries do.
Why is it that Zygote can be combined with ForwardSensitivity() and ForwardDiffSensitivity(), but ForwardDiff not with TrackerAdjoint() or QuadratureAdjoint()? This should be possible with appropriate DiffRules?

I think a note needs to be added why it does not work like that. The current note is a bit vague.

Reply:
"ForwardDiff.jl is thus not compatible with a reverse pass through, nor with continuous forward sensitivity analysis."

Reply:
Oh I see.

Reply:
The other AD libraries are able to do cross country AD, such as Zygote + ForwardDiffSensitivity(), and have the choice between discretize then optimize versus optimize then discretize. ForwardDiff does not have those options.

I guess the sentence could be confused with forward over reverse.